check/check.go

```diff
@@ -176,6 +173,11 @@ func (c *Check) Run(definedConstraints map[string][]string) {
 
 	if errmsgs != "" {
 		glog.V(2).Info(errmsgs)
+		c.Reason = out
+		// Make output more readable
+		if (errmsgs == "exit status 127" || errmsgs == "exit status 1") && strings.HasSuffix(out, "not found\n") {
+			c.Reason = strings.Replace(c.Reason, "sh: 1:", "Command", -1)
```

Comment:
Does this work in the case where the exit status is 127 and not 1, doesn't the string contain "sh: 127" in that case? I don't mind if we raise this as a separate issue so you can merge this PR fast

Reply:
I think that should be a new PR I tried to find a quick fix for pretty printing the output, basically every output on my ubuntu machine contained the sh 1:  which I thought will be the case but as your commented here in your mac its just  sh: , there is also different output per Linux flavor I guess as we saw in the tests that my machine and your machine had different output.
We need to discuss if it something that we want to deal with because we don't have to

Reply:
Ok I opened issue in the subject https://github.com/aquasecurity/bench-common/issues/90
currently I leave it unchanged.
I'm aware about the problem with the tests in mac, but here in Travis its pass, so until I will take changes in according to our decision in the above PR I will leave this part the same